dags/ltv.py

```diff
@@ -118,4 +118,15 @@
     time_partitioning={"type": "DAY", "field": "submission_date"},
 )
 
-ltv_daily >> ltv_revenue_join
+ltv_normalized_view=BigQueryOperator(
```

Comment:
The only difference between this and `ltv_revenue_join` is the destination table? As in, these contain the same data? Why not copy the partition instead of rerunning the job?

Reply:
Hah, I forgot to update the sql itself. Thanks for pointing this out.